spec/models/comment_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  author_uno = User.new(name: 'Juan', photo: 'http://jun.png', bio: 'Engineer', posts_counter: 5)
+  post_uno = Post.new(author: author_uno, title: 'First post', text: 'Lorem Ipsum', comments_counter: 2,
+                      likes_counter: 0)
+
+  context 'When testing update_like_counter method' do
+    it 'should update the post likesCounter after create a like' do
+      Like.create(post: post_uno, author: author_uno)
+      expect(post_uno.comments_counter).to eq 2
+    end
+  end
+end
```

Comment:
- [ ] Always double-check your tests! Here you're testing your `Comment#update_comments_counter` method on Comment creation, but you're creating a new `Like`, this passes because you're initializing the counter as `2`, this is a false positive test that doesn't provide any value or confidence to your code! 🤓 🔧